volumes/provisioner/jiva/snapshot_list.go

```diff
@@ -0,0 +1,188 @@
+package jiva
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/openebs/maya/command"
+)
+
+func ListSnapshot(name string) error {
+	annotations, err := command.GetVolumeSpec(name)
+	if err != nil || annotations == nil {
+
+		return err
+	}
+	controller, err := command.NewControllerClient(annotations.ControllerIP + ":9501")
+
+	if err != nil {
+		return err
+	}
+
+	replicas, err := controller.ListReplicas(controller.Address)
+	if err != nil {
+		return err
+	}
+
+	first := true
+	//snapshots := []string{}
+	for _, r := range replicas {
+		if r.Mode != "RW" {
+			continue
+		}
+
+		/*	if first {
+					first = false
+					chain, err := getChain(r.Address)
+					if err != nil {
+						return err
+					}
+					// Replica can just started and haven't prepare the head
+					// file yet
+					if len(chain) == 0 {
+						break
+					}
+					snapshots = chain[1:]
+					continue
+				}
+
+				chain, err := getChain(r.Address)
+				if err != nil {
+					return err
+				}
+
+			snapshots = Filter(snapshots, func(i string) bool {
+					return Contains(chain, i)
+				})
+				}
+
+			/*format := "%s\n"
+			tw := tabwriter.NewWriter(os.Stdout, 0, 20, 1, ' ', 0)
+			fmt.Fprintf(tw, format, "Snapshot_Name")
+			for _, s := range snapshots {
+				s = strings.TrimSuffix(strings.TrimPrefix(s, "volume-snap-"), ".img")
+				fmt.Fprintf(tw, format, s)
+			}
+			tw.Flush()
+		*/
+
+		//for _, r := range replicas {
+		if first {
+			first = false
+			snapdisk, err := getData(r.Address)
+			if err != err {
+				return err
+			}
+			out := make([]string, len(snapdisk)+1)
+
+			out[0] = "Name|Created At|Size"
```

Comment:
Use templating

Reply:
This is, to result the output in tabular form which is more similar to kubectl way of doing.
anyways will try with template too and see.